### ISSUE TITLE
CoDe Academy - City-specific subpages

### ISSUE THREAD

**Issue by adamhenriques** [Praqma/code-conf.com#94]
In order to be able to display city specific agendas, sponsors, tickets etc i suggest we create four sub pages - 1 for each of cities. 

A solution could be four pictures 1 from each city in the wound shape like we already have (see pic) - where people can chose the city they are interested in.

Subpages could be one-pagers - with same structure as our code conference sites - each with the following features as a minimum;

-Title
- Location
- Date
- Venue
- Agenda
- Sponors (Academy sponsors & Technology Sponsors)
- Ticket widget

So subpages on;

Copenhagen: http://www.code-conf.com/academy2016/cph 
Trondheim: http://www.code-conf.com/academy2016/trd
Oslo:http://www.code-conf.com/academy2016/osl
Aarhus: http://www.code-conf.com/academy2016/aar

Eventbrite id for country specific ticket widget;

Copenhagen: 24615294007
Aarhus: 24617023179
Oslo: 24616072335
Trondheim: 24573031599

![screen shot 2016-04-13 at 12 26 28](https://cloud.githubusercontent.com/assets/8004259/14490411/fd7377d4-0172-11e6-8f97-5b201eff23ef.png)

**Comment by illus0r** [comment]
@adamhenriques great, let's do this.

Please correct me if I'm wrong. 

Some events are going to be city-related. Not now, but it looks possible in the future. If so, we will keep creating 4 pages for each multi-city event (for Copenhagen, Trondheim, Oslo, Aarhus)

![image](https://cloud.githubusercontent.com/assets/797993/14494486/bc4b4522-0193-11e6-95fd-450a28e2980d.png)

There's a problem: form city related event user will not be able to find other events near his city. Each time he will have to do: 

1. go to [main index page](http://code-conf.com/);
1. go to event he need;
1. go to city related page.

We could make navigation between same-city events more easy. Just make separate index pages for each city. Once chosen a city I can browse all events around. On [main index page](http://code-conf.com/) I'll be able to browse all events of all cities at once

![image](https://cloud.githubusercontent.com/assets/797993/14494588/3e80cc06-0194-11e6-8068-f34b6db50533.png)

What do you think?

**Comment by illus0r** [comment]
we currently have no field for city. We could use title though: if it contains `CPH` it will be treated like Copenhagen  event and so on. Will be there some titles without city code inside?

**Comment by adamhenriques** [comment]
@illus0r Its only [CoDe Academy](http://www.code-conf.com/academy2016/) that will have country specific sub pages. Thus the branching should be possible once entered CoDe Academy [CoDe Academy](http://www.code-conf.com/academy2016/). - as shoved on pic 1 - but only with subpages on 1 event.  Does this make sense? If not - lets jump on a hangout

**Comment by illus0r** [comment]
@adamhenriques good, let's hangout!

**Comment by illus0r** [comment]
zor667@gmail.com

**Comment by adamhenriques** [comment]
@illus0r Thanks for the talk. If you draft the 4 subpages ill get the information you need.

**Comment by illus0r** [comment]
@adamhenriques hangout's brief
---

* We need move all city-related **Academy** information to separate city-related pages. This info includes:

1. Location
2. Date
3. Place
4. Agenda
5. Sponors
6. Ticket widget

* Leave common information on **Academy page**.
* Append city-pages links to the bottom of the **Academy page**.
* We'll reuse event template for city-related pages.
* @adamhenriques will supply some real information for city-related pages. **Especially I need at least one agenda.**
* Other website structure is untouchable

**Comment by adamhenriques** [comment]
YES:)

**Comment by illus0r** [comment]
@adamhenriques when can you send the agenda?

**Comment by adamhenriques** [comment]
@illus0r The agenda is not set yet - let me try to get a format ready for you asap @lakruzz @meekrosoft

**Comment by illus0r** [comment]
I'll assign this to you, @adamhenriques I need an agenda to go on.

**Comment by lakruzz** [comment]
```
Monday: Agile task management and Test Driven Development
  Monday evening: Teach-the-teacher workshop (Academy staff only)
Tuesday: Jenkins
Wednesday: Git
Thursday: Docker
Friday: Synthesis - bring it all together in running code.
  Friday evening: Dinner, participants meets sponsors - get a job!!

Each day at lunch time one of the sponsors will give a 20 minute pitch to the students; why they are interested in hiring.
```

**Comment by illus0r** [comment]
@lakruzz wow, that's quite different from 
![image](https://cloud.githubusercontent.com/assets/797993/14560902/5e12b196-031b-11e6-884b-b44271d7de4a.png)

I'll suggest some mockups first

**Comment by lakruzz** [comment]
No - it essentially the same as:

![image](https://cloud.githubusercontent.com/assets/155492/14561787/0f3f69da-0319-11e6-9729-4a837fc70bac.png)

**Comment by lakruzz** [comment]
I just need something online - if necessary then _without_ the agenda.

The fact that we split what we have into 4 subpages is more important than allowing ourselves to be blocked by the things we don't have

Finalizing with _Location, date, place_ and _tickets_ should be a wrap-up.

**Comment by illus0r** [comment]
@lakruzz good, I can push come city pages to start collecting info onto them.

**Comment by illus0r** [comment]
@lakruzz `http://www.code-conf.com/academy2016/cph/` is editable here

![image](https://cloud.githubusercontent.com/assets/797993/14562943/1eadd7c6-0328-11e6-85ae-d48578eb9684.png)

You can create other city-related pages in this folder

**Comment by lakruzz** [comment]
@illus0r 

You should try to avoid delivering more than one commit at a time.

![image](https://cloud.githubusercontent.com/assets/155492/14579518/5a683074-03ad-11e6-877d-09a01efabea7.png)

1) When committing _again_ upon an existing commit in the same branch use `commit --amend`
2) When synchronizing with your origin target use `pull --rebase`

**Comment by lakruzz** [comment]
@illus0r 
Closing this issue as part of #99, we have some technical debt related to this one which we will have to payback - but I'll create separate cases for them.
